controllers/glanceapi_controller.go

```diff
@@ -616,3 +622,12 @@ func (r *GlanceAPIReconciler) createHashOfInputHashes(
 	}
 	return hash, changed, nil
 }
+
+func (r *GlanceAPIReconciler) applyDefaults(instance *glancev1.GlanceAPI) bool {
```

Comment:
Looks like we're going to call this function within the `Reconcile`, actively trying to enforce the `ContainerImageURL`, while using webhooks we just react on a change to the CR, which is more efficient.